### PR TITLE
#28 changing the assignment description

### LINKED ISSUE #28: Change the assignment Description when creating an assignment
Change the text to
`This assignment was generated by the Center for IPE to track student attainment of IPE competencies. It does not impact students' grades. So please do not modify or remove it.  If you have questions, please contact IPECenter@umich.edu.`

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #28 . I have created the assignment for this courses https://canvas-test.it.umich.edu/courses/400814/assignments/1653884

This is how it will look
Screen Shot 2022-01-19 at 9 18 05 AM

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #IPE assignment naming convention:
 ASSIGNMENT_NAME = 'IPE Competencies Attained'
 ASSIGNMENT_GROUP_NAME ='IPE Competencies'
+ASSIGNMENT_DESCRIPTION = 'This assignment was generated by the Center for IPE to track student attainment of IPE competencies. It does not impact students\' grades. So please do not modify or remove it. If you have questions, please contact IPECenter@umich.edu.'
```
Comment (zqian) [inline constants.py:4]: Maybe use the full official name for IPE center as [the Michigan Center for Interprofessional Education](https://interprofessional.umich.edu/the-center/directors-welcome/)

Comment (zqian) [reply constants.py:4]: Change "So please do not modify or remove it."  to "Please do not modify or remove this assignment. ". Maybe the latter sounds more formal?

**Review by zqian — APPROVED** [review]
Just minor wording suggestions.

File: constants.py (hunk 1 above)
Comment (pushyamig) [reply constants.py:4]: That Description is provided by IPE Leads teams. So @jennlove-um thoughts? for changes

Comment (zqian) [reply constants.py:4]: Then don't worry since this is provided by IPE leads team.

**Pull request merged**